src/SDKs/DataLake.Store/DataLakeStore.Tests/DataTransferTests/DataLakeStoreClientTests.cs

```diff
@@ -388,7 +388,7 @@ public void DataLakeUploader_ResumeUploadDownloadWithAllMissingFiles()
         /// <summary>
         /// Tests the resume upload when only some segments were uploaded previously with progress tracking enabled
         /// </summary>
-        [Fact]
+        [Fact(Skip = "Randomly failing on CI. Needs investigation from DataLakeStore team")]
```

Comment:
@begoldsm If you plan to fix this in the next month, great.   Please file an issue and reference the issue in the skip message.  Otherwise, please delete this test